### PR TITLE
{vis}[GCCcore/7.3.0] tesseract v4.0.0, Leptonica v1.77.0

### PR DESCRIPTION
**Pull request by wpoely86**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic2 - Linux centos linux 7.5.1804, AMD Opteron(tm) Processor 6134, Python 2.7.5
See https://gist.github.com/2d37a46454285f74ad4d0224ea45b213 for a full test report.

**Comment by wpoely86** [comment]
Test report by @wpoely86
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
nic1 - Linux centos linux 7.6.1810, AMD Opteron(tm) Processor 6134, Python 2.7.5
See https://gist.github.com/2890242ea0507cf688fa6ea407d1e40b for a full test report.

File: easybuild/easyconfigs/l/Leptonica/Leptonica-1.77.0-GCCcore-7.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+easyblock = 'ConfigureMake'
+
+name = 'Leptonica'
+version = '1.77.0'
+
+homepage = 'http://www.leptonica.org'
+description = """Leptonica is a pedagogically-oriented open source site containing software
+ that is broadly useful for image processing and image analysis applications."""
```
Comment (akesandgren) [inline easybuild/easyconfigs/l/Leptonica/Leptonica-1.77.0-GCCcore-7.3.0.eb]: This description talks about the site, not what the Leptonica code actually does.
Please change to something relevant for the software itself.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
b-an03.hpc2n.umu.se - Linux ubuntu 16.04, Intel(R) Xeon(R) CPU E5-2690 v4 @ 2.60GHz, Python 2.7.12
See https://gist.github.com/3319d65c3f7ac543e5bd4ae975edc1e5 for a full test report.

File: easybuild/easyconfigs/l/Leptonica/Leptonica-1.77.0-GCCcore-7.3.0.eb (hunk 1 above)
Comment (wpoely86) [reply easybuild/easyconfigs/l/Leptonica/Leptonica-1.77.0-GCCcore-7.3.0.eb]: good point, fixed

**Comment by wpoely86** [comment]
Green!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node3116.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 2.7.5
See https://gist.github.com/95f0a2d329f49c5e3c81b92c54227685 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
node2543.golett.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 2.7.5
See https://gist.github.com/d00bbf80ca0d27f43a803dc315fe6b91 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @wpoely86!

**Pull request merged**